### DISCUSSION TITLE
如何确定我是否在使用XTLS?

### DISCUSSION THREAD

**Discussion by Alceatraz** [XTLS/Xray-core#2803] [General]
RT，一个比较蠢的问题：

我按照教程和示例配置了 vless + xtls-rprx0-vision 但是streamSetting依然是TCP+TLS+CRT

xtls本来就应该这样配置吗?他是vless流控而非streamSetting?因为看到解释说他能选择性避免二次TLS，我觉得是不是应该在streamSetting的才对？在example中看到过security=xtls的设置

**Comment by chise0713** [comment]
那个example太老了，添在flow里是对的，可以把logLevel调为info查看xtls调试信息
